lib/android/app/src/test/java/com/reactnativenavigation/viewcontrollers/ContainerViewControllerTest.java

```diff
@@ -126,4 +131,35 @@ public void appliesTopBackBackgroundColor() throws Exception {
 
 		assertThat(((ColorDrawable) stackController.getTopBar().getBackground()).getColor()).isEqualTo(0xffff0000);
 	}
+
+	@Test
+	public void appliesTopBarTextColor() throws Exception {
+	  assertThat(uut.getNavigationOptions()).isSameAs(initialNavigationOptions);
+	  initialNavigationOptions.title = "the title";
+	  StackController stackController = new StackController(activity, "stackId");
+	  stackController.push(uut);
+	  uut.onViewAppeared();
+	  assertThat(getTitleTextView(stackController.getTopBar().getToolbar()).getCurrentTextColor()).isNotEqualTo(Color.RED);
+
+	  NavigationOptions opts = new NavigationOptions();
+	  opts.topBarTextColor = Color.RED;
+	  uut.mergeNavigationOptions(opts);
+
+	  assertThat(getTitleTextView(stackController.getTopBar().getToolbar()).getCurrentTextColor()).isEqualTo(Color.RED);
+	}
+
+  private TextView getTitleTextView(Toolbar toolbar) {
```

Comment:
please find a way to test this without going through reflection. Maybe robolectric can assist you..